segment_path.go

```diff
@@ -64,11 +64,27 @@ func (pt *path) getShortPath() string {
 	if strings.HasPrefix(pwd, "Microsoft.PowerShell.Core\\FileSystem::") {
 		pwd = strings.Replace(pwd, "Microsoft.PowerShell.Core\\FileSystem::", "", 1)
 	}
+
 	mappedLocations := map[string]string{
 		"HKCU:":          pt.props.getString(WindowsRegistryIcon, "\uE0B1"),
 		"HKLM:":          pt.props.getString(WindowsRegistryIcon, "\uE0B1"),
 		pt.env.homeDir(): pt.props.getString(HomeIcon, "~"),
 	}
+
+	// merge custom locations with mapped locations
+	// mapped locations can override predefined locations
+	if pt.props.values["mappedlocations"] != nil {
```

Comment:
This needs to be a function in `properties.go` just like [here](https://github.com/JanDeDobbeleer/oh-my-posh3/blob/main/properties.go#L88) for example. Then we can use `append` to merge the maps (I believe).